www/src/components/plugin-searchbar-body.js

```diff
@@ -303,9 +303,11 @@ class Search extends Component {
 // the result component is fed into the InfiniteHits component
 const Result = ({ hit, pathname, query }) => {
   // Example:
-  // pathname = `/plugins/gatsby-link/` || `/plugins/@comsoc/gatsby-mdast-copy-linked-files`
+  // pathname = `/packages/gatsby-link/` || `/packages/@comsoc/gatsby-mdast-copy-linked-files`
   //  hit.name = `gatsby-link` || `@comsoc/gatsby-mdast-copy-linked-files`
-  const selected = pathname.includes(hit.name)
+  const packagesRegex = new RegExp(`/packages/`, `g`)
+  const packageName = pathname.replace(packagesRegex, ``).slice(0, -1)
+  const selected = packageName === hit.name
```

Comment:
```suggestion
  const selected = new RegExp(`^/packages/${hit.name}/?$`).test(pathname)
```
we could replace it with above regex. IMO it's clearer than the 3 lines.

Reply:
@wardpeet I like this too :+1: